modules/tinymce/src/core/main/ts/api/dom/BookmarkManager.ts

```diff
@@ -66,18 +66,16 @@ function BookmarkManager(selection: EditorSelection): BookmarkManager {
      */
     moveToBookmark: Fun.curry(Bookmarks.moveToBookmark, selection) as (bookmark: Bookmark) => boolean
   };
-}
+};
 
-namespace BookmarkManager {
-  /**
-   * Returns true/false if the specified node is a bookmark node or not.
-   *
-   * @static
-   * @method isBookmarkNode
-   * @param {DOMNode} node DOM Node to check if it's a bookmark node or not.
-   * @return {Boolean} true/false if the node is a bookmark node or not.
-   */
-  export const isBookmarkNode = Bookmarks.isBookmarkNode as (node: Node) => boolean;
-}
+/**
+ * Returns true/false if the specified node is a bookmark node or not.
+ *
+ * @static
+ * @method isBookmarkNode
+ * @param {DOMNode} node DOM Node to check if it's a bookmark node or not.
+ * @return {Boolean} true/false if the node is a bookmark node or not.
+ */
+BookmarkManager.isBookmarkNode = Bookmarks.isBookmarkNode as (node: Node) => boolean;
```

Comment:
Wondering if the type assertion is required here?

Reply:
It was like this previously, so I'm hesitant to remove it. It shouldn't be needed though...